Resources/Locale/en-US/actions/actions/uncuff-self.ftl

```diff
@@ -0,0 +1,2 @@
+action-name-uncuff-self = Uncuff Self
+action-description-uncuff-self = Toggle attempting to uncuff yourself.
```

Comment:
Put these in `Resources/Locale/en-US/cuffs/components/cuffable-component.ftl` with the rest, and change `cuffable-component-start-uncuffing-self` to no longer refer to pain/damage

Reply:
File has been yeeted in favor of doing the cooldown via UseDelay; there's no longer any Action button that requires extra text.